site/sfguides/src/getting_started_with_native_apps/getting_started_with_native_apps.md

```diff
@@ -314,233 +405,111 @@ grant usage on streamlit app_instance_schema.streamlit to application role app_i
 ```
 
-This flow ensures that the data is able to be shared securely with the consumer through the application. The objects containing the provider's proprietary shipping data are **never** shared directly with the consumer via a Snowflake Native Application. This means the provider's proprietary data remains safe, secure, and in the provider's Snowflake account. Instead, the application package has reference usage on objects (databases) corresponding to the provider's data, and when a consumer install the app (i.e., instantiates the application package), they are able to use the shared data through the application.
-
 <!-- ------------------------ -->
 ## Create App Package Version
 Duration: 4
 
-We've now uploaded the application source code to our stage, created our application package, and uploaded the provider data to be shared in the application package.
-
-From here, we will create the first version of our application package. You can have multiple versions available. 
-
-To view our App Package in the Snowflake UI:
-
-1. Login to your Snowflake account and navigate to `Apps`
-2. In the top navigation, click `Packages`
-3. Click `NATIVE_APP_QUICKSTART_PACKAGE`
-
-Now let's add our first version using our previously uploaded code.
-
-1. Click `Add first version`.
-2. Type `V1` for the version name.
-3. Select `NATIVE_APP_QUICKSTART_PACKAGE` for the database.
-4. Select `NATIVE_APP_QUICKSTART_STAGE` for the stage.
-5. Select `/` for the directory.
-6. Create the first version.
-
-<!-- ------------------------ -->
-## Upload Consumer Supply Chain Data
-Duration: 2
-
-In this scenario, consumers will provider their own supply chain data (orders and site recovery data) from their own Snowflake account. The app will use the consumer's data to render graphs representing different aspects of the supply chain.
+We've now added some basic functionality to the native app. From here, we will create the first version of our application package. You can have multiple versions available. 
 
-Run the SQL below in a worksheet. We'll use the `NATIVE_APP_QUICKSTART_DB` to store the consumer supply chain data.
+Yo create our application package version we are going to execute the following command:
 
 ```
-USE WAREHOUSE NATIVE_APP_QUICKSTART_WH;
-
--- this database is used to store our data
-USE DATABASE NATIVE_APP_QUICKSTART_DB;
-
-USE SCHEMA NATIVE_APP_QUICKSTART_SCHEMA;
-
-CREATE OR REPLACE TABLE MFG_ORDERS (
-  order_id NUMBER(38,0), 
-  material_name VARCHAR(60),
-  supplier_name VARCHAR(60),
-  quantity NUMBER(38,0),
-  cost FLOAT,
-  process_supply_day NUMBER(38,0)
-);
-
--- Load app/data/orders_data.csv using Snowsight
-
-CREATE OR REPLACE TABLE MFG_SITE_RECOVERY (
-  event_id NUMBER(38,0), 
-  recovery_weeks NUMBER(38,0),
-  lat FLOAT,
-  lon FLOAT
-);
-
--- Load app/data/site_recovery_data.csv using Snowsight
+snow app version create V1 --interactive --database NATIVE_APP_QUICKSTART_PACKAGE
```

Comment:
Is the `--database` necessary here?

Reply:
You are right, I just tested it and it is not necessary